README.md

```diff
@@ -13,6 +13,41 @@ pip install cdsetool
 
 ## Usage
 
+### Quick start
```

Comment:
Make the "Quick start" section a H2 (from `###` to `##`) and move it above `## Installation`